share/commands/agent-check

```diff
@@ -103,6 +101,22 @@ check_and_fix_cfengine_keys() {
   fi
 }
 
+# There must be a policy server otherwise we can't do anything
+check_policy_server_or_exit() {
+  if [ ! -s ${CFE_DIR}/policy_server.dat ]
+  then
+    echo "********************************************************************************"
+    echo "rudder-agent has been installed (not started). This host can be a Rudder node."
+    echo "To get started, configure your Rudder server's hostname and continue the process:"
+    echo "# echo 'rudder.server' > ${CFE_DIR}/policy_server.dat"
```

Comment:
we should display /opt/rudder/etc instead here

Reply:
I think this is the lase place that should be changed since if the symlink was not created this will fail